extensions/geode-modules-tomcat8/src/distributedTest/java/org/apache/geode/modules/session/TestSessionsTomcat8Base.java

```diff
@@ -399,7 +407,7 @@ public void testExtraSessionsNotCreated() throws Exception {
 
     WebResponse response = wc.getResponse(req);
     assertThat(response.getText()).isEqualTo("done");
-    assertThat(region.size()).as("The region should be empty").isEqualTo(0);
+    assertThat(region.size()).as("The region should be empty").isEqualTo(1);
```

Comment:
This assertion says `"The region should be empty"` but then it's asserted to be `equalTo(1)` which isn't empty. Should it be empty or one?

Reply:
fixed.